### ISSUE TITLE
think about preference elicitation in ScPo context

### ISSUE THREAD

**Issue by floswald** [ScPo-CompEcon/CourseMatch.jl#3]
at wharton:

1. choose utility 0-100 for each course
1. can choose pairwise adjustment for courses that are preferrably taken together (choose 200 if very preferrable to take together, choose -200 if not preferrable at all)

do we need that? what else do we need that is specific to ScPo? may need to look at the ScPo report on coursematch for this.

**Comment by JulieLen** [comment]
I completely agree to that, the problem can be broken down into smaller problems, which will make our job easier. 
I was wondering if we should allow every student to express preferences on each and every couple.


Moreover I was asking myself a question : 
- Should we allow student to express preferences on each and every couple of class ? If we do, should we allow student to express preferences only on couples of electives, or can they also express the fact that they want this "Tronc commun" class and this elective ? 
- Or should we limit the number of cross-preferences they can express ?

**Comment by dsl55** [comment]
Good question concerning how many couples a student should be allowed to define. In general, I'd say that a richer set of information will yield a better fit for the student in question. However, one concern could be that students spend a lot of time on defining a lot of couples. To save them from this work, one could limit it.

Regarding the second question related to cross-complementaries between Tronc commun and CC: I think that the value added of it is marginal and it makes things more complicated.
